jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/ReservedThreadExecutor.java

```diff
@@ -87,7 +57,7 @@ public String toString()
     public ReservedThreadExecutor(Executor executor, int capacity)
     {
         _executor = executor;
-        _capacity = reservedThreads(executor, capacity);
+        _threads = new ThreadIdPool<>(reservedThreads(executor, capacity));
```

Comment:
Add `_threads` as a bean, either here, or in `doStart()`.

Reply:
@sbordet hey review #11319 so I can correctly do an `installBean` here rather than an `addBean`!!!